test/JDBC/expected/BABEL-3201-vu-verify.out

```diff
@@ -0,0 +1,59 @@
+DBCC CHECKIDENT(babel_3201_t1);
```

Comment:
Please also add some tests to validate transactional behaviour.
Running DBCC inside a transaction
DBCC running into error inside transaction
Calling DBCC from procedure/functions (with/without errors)

Reply:
Added tests for transactions and procedures. Please take a look. Thanks.